src/common/camera_control.c

```diff
@@ -26,10 +26,7 @@
 
 #include <unistd.h>
 #include <stdlib.h>
-#if defined(__SUNOS__)
```

Comment:
Am I correct in assuming that you believe the #include <fcntl.h> should be valid for every OS which dt builds on? How many people running dt on the non-linux OSes have you confirmed this with?